Segfault with simulcast track add / remove (possible race)

### Your environment.
  - Version: master
  - Browser: Electron 9 / Chromium
### What did you do?
Using ion-sfu, I add a simulcast track to an existing stream w/ audio.  I then remove the track prior to all RID layers being received by the sfu, which triggers a segfault.

### What did you expect?
not crash

### What happened?

Segfault
```
[2020-10-08 16:30:01.706] [DEBUG] [78][webrtctransport.go][func1] => Peer ckg1ge6js00011l339dm78azw got remote track id: 59d2fa89-25d5-4eab-a571-222865c2502a ssrc: 422920037 rid : streamID: eg3vuEezE548pcKNE8hC2Gu8rOjxCPbpltSr
[2020-10-08 16:30:01.706] [INFO] [100][webrtctransport.go][func1] => AddRouter ssrc to ckg1ge50800001l33copxodwo
[2020-10-08 16:30:01.707] [DEBUG] [104][webrtctransport.go][func1] => Created router ckg1ge6js00011l339dm78azw 422920037
[2020-10-08 16:30:01.707] [DEBUG] [316][peerconnection.go][negotiationNeededOp] => on negotiation needed called
[2020-10-08 16:30:01.757] [INFO] [320][main.go][Handle] => peer ckg1ge50800001l33copxodwo answer
[2020-10-08 16:30:03.981] [INFO] [264][main.go][Handle] => peer ckg1ge50800001l33copxodwo offer
[2020-10-08 16:30:04.008] [DEBUG] [339][main.go][Handle] => trickle
[2020-10-08 16:30:04.008] [INFO] [349][main.go][Handle] => peer ckg1ge50800001l33copxodwo trickle
[2020-10-08 16:30:05.442] [DEBUG] [78][webrtctransport.go][func1] => Peer ckg1ge50800001l33copxodwo got remote track id: 414bedca-7a83-4142-87af-139e5dc212c4 ssrc: 4093586747 rid :f streamID: 79Yq3EICl0KxIiG3NXrq4E8eerBFlTaWQdmq
[2020-10-08 16:30:05.443] [DEBUG] [416][receiver.go][startVideoReceiver] => NewBuffer BufferOptions={1000}
[2020-10-08 16:30:05.443] [DEBUG] [429][receiver.go][startVideoReceiver] => Setting feedback goog-remb
[2020-10-08 16:30:05.443] [INFO] [100][webrtctransport.go][func1] => AddRouter ssrc to ckg1ge6js00011l339dm78azw
[2020-10-08 16:30:05.443] [DEBUG] [104][webrtctransport.go][func1] => Created router ckg1ge50800001l33copxodwo 4093586747
[2020-10-08 16:30:05.443] [DEBUG] [316][peerconnection.go][negotiationNeededOp] => on negotiation needed called
[2020-10-08 16:30:05.453] [DEBUG] [78][webrtctransport.go][func1] => Peer ckg1ge50800001l33copxodwo got remote track id: 414bedca-7a83-4142-87af-139e5dc212c4 ssrc: 3021681678 rid :q streamID: 79Yq3EICl0KxIiG3NXrq4E8eerBFlTaWQdmq
[2020-10-08 16:30:05.453] [DEBUG] [416][receiver.go][startVideoReceiver] => NewBuffer BufferOptions={1000}
[2020-10-08 16:30:05.453] [DEBUG] [429][receiver.go][startVideoReceiver] => Setting feedback goog-remb
[2020-10-08 16:30:05.503] [INFO] [320][main.go][Handle] => peer ckg1ge6js00011l339dm78azw answer
[2020-10-08 16:30:05.916] [INFO] [264][main.go][Handle] => peer ckg1ge50800001l33copxodwo offer
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1360722]

goroutine 18 [running]:
github.com/pion/srtp.(*ReadStreamSRTCP).Close(0x0, 0x0, 0x0)
        /Users/billy/Development/go/pkg/mod/github.com/pion/srtp@v1.5.2/stream_srtcp.go:66 +0x42
github.com/pion/webrtc/v3.(*RTPReceiver).Stop(0xc00020a900, 0x0, 0x0)
        /Users/billy/Development/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.7.0.20201008225834-7b72d684e541/rtpreceiver.go:170 +0x117
github.com/pion/webrtc/v3.(*RTPTransceiver).Stop(0xc0002005a0, 0x1, 0xc0001b8610)
        /Users/billy/Development/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.7.0.20201008225834-7b72d684e541/rtptransceiver.go:81 +0x17b
github.com/pion/webrtc/v3.(*PeerConnection).SetRemoteDescription(0xc000166000, 0x1, 0xc0001ee800, 0x1594, 0x0, 0x0, 0x0)
        /Users/billy/Development/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.7.0.20201008225834-7b72d684e541/peerconnection.go:998 +0x105e
github.com/pion/ion-sfu/pkg.(*WebRTCTransport).SetRemoteDescription(0xc00014a1e0, 0x1, 0xc0001ee800, 0x1594, 0x0, 0x0, 0x0)
        /Users/billy/Development/go/src/github.com/pion/ion-sfu/pkg/webrtctransport.go:181 +0x78
main.(*RPC).Handle(0xc000010680, 0x166d100, 0xc00021a000, 0xc00021c000, 0xc0003d0af0)
        /Users/billy/Development/go/src/github.com/pion/ion-sfu/cmd/server/json-rpc/main.go:278 +0x1cc6
github.com/sourcegraph/jsonrpc2.(*Conn).readMessages(0xc00021c000, 0x166d100, 0xc00021a000)
        /Users/billy/Development/go/pkg/mod/github.com/sourcegraph/jsonrpc2@v0.0.0-20200429184054-15c2290dcb37/jsonrpc2.go:522 +0x57e
created by github.com/sourcegraph/jsonrpc2.NewConn
        /Users/billy/Development/go/pkg/mod/github.com/sourcegraph/jsonrpc2@v0.0.0-20200429184054-15c2290dcb37/jsonrpc2.go:334 +0x1d5```

--- a/README.md
+++ b/README.md
@@ -205,6 +205,7 @@ Check out the **[contributing wiki](https://github.com/pion/webrtc/wiki/Contribu
 * [Hanjun Kim](https://github.com/hallazzang)
 * [ZHENK](https://github.com/scorpionknifes)
 * [Rahul Nakre](https://github.com/rahulnakre)
+* [OrlandoCo](https://github.com/OrlandoCo)
 
 ### License
 MIT License - see [LICENSE](LICENSE) for full text

--- a/rtpreceiver.go
+++ b/rtpreceiver.go
@@ -167,11 +167,15 @@ func (r *RTPReceiver) Stop() error {
 	select {
 	case <-r.received:
 		for i := range r.tracks {
-			if err := r.tracks[i].rtcpReadStream.Close(); err != nil {
-				return err
+			if r.tracks[i].rtcpReadStream != nil {
+				if err := r.tracks[i].rtcpReadStream.Close(); err != nil {
+					return err
+				}
 			}
-			if err := r.tracks[i].rtpReadStream.Close(); err != nil {
-				return err
+			if r.tracks[i].rtpReadStream != nil {
+				if err := r.tracks[i].rtpReadStream.Close(); err != nil {
+					return err
+				}
 			}
 		}
 	default:

